homeassistant/components/elevenlabs/quality_scale.yaml

```diff
@@ -0,0 +1,85 @@
+rules:
+  # Bronze
+  action-setup:
+    status: done
+    comment: >
+      Only entity services
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions: done
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: todo
+  entity-event-setup:
+    status: exempt
+    comment: |
+      Entities of this integration does not explicitly subscribe to events.
+  entity-unique-id: done
+  has-entity-name: todo
+  runtime-data: done
+  test-before-configure: done
+  test-before-setup: done
+  unique-config-entry: todo
+  # Silver
+  config-entry-unloading: todo
+  log-when-unavailable: todo
+  entity-unavailable: todo
+  action-exceptions: done
+  reauthentication-flow: todo
+  parallel-updates: done
+  test-coverage: done
```

Comment:
```suggestion
  test-coverage: todo
```
Missing coverage

Reply:
Also, let's patch libraries where we use them, like `"elevenlabs.AsyncElevenLabs"`

Reply:
Coverage is 97% according to my pytest run

Reply:
```
---------- coverage: platform darwin, python 3.12.2-final-0 ----------
Name                                                 Stmts   Miss  Cover   Missing
----------------------------------------------------------------------------------
homeassistant/components/elevenlabs/__init__.py         40      5    88%   27, 51-52, 55, 74
homeassistant/components/elevenlabs/config_flow.py      67      0   100%
homeassistant/components/elevenlabs/const.py            15      0   100%
homeassistant/components/elevenlabs/tts.py              53      1    98%   132
----------------------------------------------------------------------------------
TOTAL                                                  175      6    97%

```
Every module should be 95%+